.circleci/config.yml

```diff
@@ -188,7 +188,7 @@ jobs:
       - run: sudo brainframe compose up -d
       - assert-core-container-running
       - assert-core-responds-to-http
-      - run: sudo brainframe compose down
+      - run: sudo brainframe uninstall --noninteractive
```

Comment:
After this, we would want to run some kind of `assert-fully-uninstalled` which just makes sure some directories don't exist

Reply:
Oh yes very true indeed

Reply:
LGTM!